client/src/visitors/pages/main.js

```diff
@@ -121,6 +122,7 @@ export default class Main extends Component {
         this.setState({
           cbOrgName: res.data.result.name,
           cbLogoUrl: res.data.result.logoUrl,
+          organisationId: res.data.result.id,
```

Comment:
This wont exist, look at the `fields`.

Reply:
Also pls update the test payload

Reply:
Weird that it works though huh? 🙃

Reply:
Lol, ok I'll make an issue in `twine-api` then.